e2e-tests/development-runtime/cypress.json

```diff
@@ -1,4 +1,5 @@
 {
   "baseUrl": "http://localhost:8000",
-  "failOnStatusCode": false
+  "failOnStatusCode": false,
+  "chromeWebSecurity": false
```

Comment:
Note: I had to enable this, otherwise the `window.location.reload()` calls in the restarting flow break Crypess. :confused:

Reply:
Crypess lol, amazing typo